src/Api/Management/Entity/Organization.php

```diff
@@ -43,6 +43,10 @@ class Organization extends Entity implements OrganizationInterface
     protected const TYPES = [
```

Comment:
Could we add a comment specifying which org type comes from Apigee Edge vs Apigee X?

Reply:
I've added the same in comment along with documentation reference.